manim/animation/fading.py

```diff
@@ -127,31 +145,35 @@ class FadeOutAndShiftDown(FadeOutAndShift):
     communicates the default
     """
 
-    def __init__(self, mobject, direction=DOWN, **kwargs):
+    def __init__(
+        self, mobject: "Mobject", direction: np.ndarray = DOWN, **kwargs
+    ) -> None:
         super().__init__(mobject, direction=direction, **kwargs)
         logger.warning(
             "FadeOutAndShiftDown is deprecated and will eventually disappear. Please use FadeOutAndShift(<mobject>, direction=DOWN, <other_args>) instead."
         )
 
 
 class FadeInFromPoint(FadeIn):
-    def __init__(self, mobject, point, **kwargs):
+    def __init__(
+        self, mobject: "Mobject", point: np.ndarray, **kwargs
+    ) -> None:  # is point typed correctly?
```

Comment:
`# is point typed correctly?`

I believe point can be either an ndarray or another mobject. Haven't tried ti tho.

Reply:
So should I add `typing.Union[Mobject,np.ndarray]`?

Reply:
if it works with a Mobject, then yes

Reply:
Does it "work"? Have you tried it?

Reply:
I have not tried it. Maybe you can try it :slightly_smiling_face:

Reply:
"worked"